src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public enum commands {
+        BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE;
+
+        @Override
+        public String toString() {
+            return name().toLowerCase();
+        }
+    }
+
+    private static String numOfTasks(ArrayList<Task> tasks) {
```

Comment:
I agree with @LiuMC-SG, perhaps this can be a suggestion?
```suggestion
    private static String getTaskString(ArrayList<Task> tasks) {
```